### PR TITLE
[CCXDEV-11589] Add clowdapp for dvo-writer

### PR DESCRIPTION
**Pull request by juandspy**
# Description

Adding a Clowdapp for running the dvo-writer.

## Type of change

- New feature (non-breaking change which adds functionality)
- Configuration update

## Testing steps

TBD

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: deploy/db-writer.yaml [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+- name: PAYLOAD_TRACKER_TOPIC
+  value: platform.payload-status
+  required: true
+- name: DVO_WRITER_SERVICE_NAME
+  value: dvo-writer
+  required: true
+- name: GROUP_ID
+  value: ccx_data_pipeline_app
+  required: true
+- name: IRDW_API_PREFIX
```
Comment (Bee-lee) [inline deploy/db-writer.yaml:270]: IRDW means insights results DB writer

Comment (juandspy) [reply deploy/db-writer.yaml:270]: oh, true, I will use another name then

Comment (juandspy) [reply deploy/db-writer.yaml:270]: fixed in https://github.com/RedHatInsights/insights-results-aggregator/pull/1906/commits/12f69fb3d9d768cb952d89d0da32138fd1023271

**Comment by juandspy** [comment]
> @juandspy we need to use this config field https://github.com/RedHatInsights/insights-results-aggregator/blob/master/config.toml#L53 to use `dvo_recommendations` so that it will create a connection for the proper interface

Thanks a lot @Bee-lee ! I totally missed that one. Should be fixed in https://github.com/RedHatInsights/insights-results-aggregator/pull/1906/commits/4afead259dc158471670219b7a8a3120c7e3eddd.

File: deploy/dvo-writer.yaml [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+- name: DVO_WRITER_SERVICE_NAME
+  value: dvo-writer
+  required: true
+- name: GROUP_ID
+  value: dvo_writer_app
+  required: true
+- name: DVO_WRITER_API_PREFIX
+  required: true
+  value: /
+- name: IRDW_LOG_STREAM
```
Comment (joselsegura) [inline deploy/dvo-writer.yaml:277]: For parameters, I would use more meaningful names instead all this acronyms that we usually use. At the end, those are "local" to the `ClowdApp`, so no need to be so pedantic :-)

**Review by joselsegura — APPROVED** [review]

File: deploy/dvo-writer.yaml (hunk 2 above)
Comment (juandspy) [reply deploy/dvo-writer.yaml]: fixed in https://github.com/RedHatInsights/insights-results-aggregator/pull/1906/commits/0fc0fb2b0eb74bcddfe361ae9105f2da427ed4c8, ty!

**Review by Bee-lee — APPROVED** [review]

**Review by tisnik — APPROVED** [review]
LGTM

**Pull request merged**